rustc_tools_util/src/lib.rs

```diff
@@ -2,19 +2,21 @@
 
 use std::env;
 
```

Comment:
do we still need the import then?

Reply:
I don't know, I would have expected a warning, but I can test it :)

Reply:
Only need to fix `match env::var("CFG_RELEASE_CHANNEL") {` then we can remove it. :+1: